src/main/java/org/assertj/core/error/ShouldHaveNoExtension.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2021 the original author or authors.
+ */
+package org.assertj.core.error;
+
+import org.assertj.core.util.VisibleForTesting;
+
+import java.nio.file.Path;
+
+/**
+ * Creates an error message indicating that a {@code Path} should have no extension.
+ */
+public class ShouldHaveNoExtension extends BasicErrorMessageFactory {
+
+  @VisibleForTesting
+  public static final String PATH_HAS_EXTENSION = "%nExpected actual path:%n  %s%n not to have an extension, but extension was:%n  %s";
```

Comment:
no need to go to the next line which is used to align values to compare usually so `was: %s"` should be enough here.

Reply:
fixed